target/linux/ath79/image/generic.mk

```diff
@@ -2117,6 +2117,19 @@ define Device/rosinson_wr818
 endef
 TARGET_DEVICES += rosinson_wr818
 
+define Device/ruckus_r500
+  SOC := qca9557
+  DEVICE_TITLE := Ruckus R500
+  IMAGE_SIZE := 25165k
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct
+  KERNEL := kernel-bin | append-dtb
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | uImage none
+  IMAGES += kernel.bin rootfs.bin
+  IMAGE/rootfs.bin := append-rootfs | check-size $$$$(IMAGE_SIZE)
```

Comment:
I do not see any use for this in the flashing instructions?

Reply:
Removed them, and added header generator instead